execution/executor/transfer.go

```diff
@@ -54,6 +54,9 @@ func (e *TransferExecutor) Execute(trx *tx.Tx, sb sandbox.Sandbox) error {
 		return errors.Errorf(errors.ErrInvalidSequence,
 			"expected: %v, got: %v", senderAcc.Sequence()+1, trx.Sequence())
 	}
+	if senderAcc.Balance() < pld.Amount+trx.Fee() {
```

Comment:
There is same logic several line above, Remove that.
The point is, first check sequence then balance.

Reply:
fixed